CMSIS/CM3/DeviceSupport/MDR32F9Qx/inc/MDR32Fx.h

```diff
@@ -3455,20 +3455,30 @@ typedef struct
 #define BITBAND_PER_REF     (0x40000000)
 #define BITBAND_PER_BASE    (0x42000000)
 
-#define BIT_RAM(a,b) *((volatile uint32_t *)(BITBAND_SRAM_BASE+((a)-BITBAND_SRAM_REF)*32+((b)*4)))
+#define BIT_RAM_P(a,b) *((volatile uint32_t *)(BITBAND_SRAM_BASE+((uint32_t)(a)-BITBAND_SRAM_REF)*32+((b)*4)))
 #define BIT_PER(a,b) *((volatile uint32_t *)(BITBAND_PER_BASE+((uint32_t)&(a)-BITBAND_PER_REF)*32+((b)*4)))
 
-#define CLR_BIT_RAM(a,b) (BIT_RAM(a,b) = 0x00000000)
-#define CLR_BIT_PER(a,b) (BIT_PER(a,b) = 0x00000000)
+#define CLR_BIT_RAM_P(a,b) do { \
+    BIT_RAM_P(a,b) = 0; \
+} while ( 0 )
```

Comment:
I don't understand for what is this?
```C
#define CLR_BIT_RAM_P(a,b) do { \
    BIT_RAM_P(a,b) = 0; \
} while ( 0 )
```
And why you change library API. You should use like this BIT_RAM(**p**a,b) instead BIT_RAM_P(a,b).